README.md

```diff
@@ -52,7 +52,7 @@ Each author record has the fields:
 * id (self-generated)
```

Comment:
Eu mudaria a frase acima daqui para `Each author record in the database has the fields:`, acho que fica mais claro...

Reply:
Que tal `Each author record in the database should has the fields`?

Reply:
Volta aqui, @felipegr. ahahah

Reply:
Acho que poderia ser então `Each author record in the database must have the fields`, que acha?

Reply:
Ótemo!